package.json

```diff
@@ -2170,6 +2170,7 @@
         "iconv-lite": "^0.4.21",
         "image-size": "^1.0.0",
         "inversify": "^5.0.1",
+        "is-localhost-ip": "^1.4.0",
```

Comment:
The npmjs entry actually said GPL, but from the actual repro it looks like it was licensed to MIT years ago. I'd assume that's ok? https://github.com/tinovyatkin/is-localhost-ip/commit/f9172384513ddca3245c460dbf66ef5234e8311c

Reply:
Wondering if component governance uses npm entry...

Reply:
I guess we'll find out :)